src/Application/Messenger/Handler/SendRequestConsumer.php

```diff
@@ -56,27 +56,36 @@ public function __invoke(SendRequest $sendRequest): void
             $this->fail($sendRequest->id, "Validation failed: {$this->validator->getReason()}");
         }
 
-        $bodyRenderedHtml = $this->twig->render("{$sendRequest->templateKey}.html.twig", $sendRequest->params);
+        // Instantiate a new Swift Message object
+        $email = new \Swift_Message();
+
+        $pathToImages = __DIR__ . '/../../../../images/';
+
+        $images['header'] = $email->embed(\Swift_Image::fromPath($pathToImages . 'TBG.jpg'));
```

Comment:
These get mushed in with the other params, so this naming is not very maintainable. How about `headerImageRef` etc. or an array key `imageRefs` with these as inner array elements?

As mentioned on our call I think this boilerplate would also be good to abstract out to an isolated helper method. At the least we should avoid doing all the same path gubbins as a copy/paste.